virtdata-lib-curves4/src/main/java/io/nosqlbench/virtdata/library/curves4/continuous/int_double/ConstantContinuous.java

```diff
@@ -32,6 +32,6 @@
 @Categories({Category.distributions})
 public class ConstantContinuous extends IntToDoubleContinuousCurve {
     public ConstantContinuous(double value, String... mods) {
-        super(new ConstantContinuousDistribution(value), mods);
```

Comment:
I can't find the docs on ConstantContinuousDistribution anymore on apache commons:
https://commons.apache.org/proper/commons-statistics/commons-statistics-distribution/apidocs/org/apache/commons/statistics/distribution/ConstantContinuousDistribution.html
Are we sure the EmpiricalDistribution can execute the same role?

Reply:
The implementation of the current form is effectively hidden in a private implementation behind the EmpiricalDistribution, so we are just using a different way to get it.